File: Company/src/main/java/company/service/CompanyService.java
package company.service;

import java.util.List;

import company.bean.Firma;
import company.bean.RacunFirme;

public interface CompanyService {

	List<Firma> getCompanies();

	Firma getThisCompany();

	List<RacunFirme> getThisCompanyBills();

}

File: Company/src/main/resources/static/Payment Order/CompanyService.js
paymentOrderModule.service('companyService', function() {

	this.getThisCompany = function() {
		return $http({
			method : 'GET',
			url : '../Firma/company/this-company',
			headers : {
				'Accept' : 'application/json'
			}
		}).then(function successCallback(response) {
			return response;
		}, function errorCallback(response) {
			return response;
		});
	}

});